spec/domain/people/duplicate_checker_spec.rb

```diff
@@ -0,0 +1,69 @@
+require 'spec_helper'
+
+describe People::DuplicateChecker do
+
+  let(:checker) { described_class.new }
+  subject { checker.check }
+
+  context 'check' do
+    it 'creates one duplicate when all attributes match once' do
+      create_duplicate(:top_leader)
+      expect { subject }.to change { PersonDuplicate.count }.by(1)
```

Comment:
yay, you discovered the subject pattern for rspec.
but ... I think it's better to use just checker.check in this case. it makes it more readable. 

subject is pretty good if you use for example the #its methods for checking an objects values.